src/features/certificateManager/CertificateManager.tsx

```diff
@@ -220,7 +220,7 @@ export default ({ active }: { active: boolean }) => {
     };
 
     const className =
-        'cert-mgr-view d-flex flex-column p-4 h-100 overflow-auto pretty-scrollbar';
+        'd-flex flex-column p-4 h-100 tw-overflow-y-scroll styled-scroll';
```

Comment:
I think we use overflow-y: overlay for our scrollbar. Am a bit unsure now after seeing this but if it's not that different, I'd prefer to keep the same convention.